src/c++/perf_analyzer/docs/llm.md

```diff
@@ -31,9 +31,55 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 The following guide shows the reader how to use Triton
 [Perf Analyzer](https://github.com/triton-inference-server/client/tree/main/src/c%2B%2B/perf_analyzer)
 to measure and characterize the performance behaviors of Large Language Models
-(LLMs) using Triton with [vLLM](https://github.com/vllm-project/vllm).
+(LLMs) using Triton with [TensorRT-LLM](https://github.com/NVIDIA/TensorRT-LLM) and [vLLM](https://github.com/vllm-project/vllm).
 
-### Setup: Download and configure Triton vLLM Backend
+## Setup: Download and configure Triton Server environment
+
+### Using TensorRT-LLM
+
+1. Follow [step 1](https://github.com/triton-inference-server/tutorials/blob/main/Popular_Models_Guide/Llama2/trtllm_guide.md#installation) of the Installation section.
+
+    This guide will replace the llama model for gpt since it is already included in the tensorrtllm_backend repository release.
+
+2. Launch the Triton docker container with the TensorRT-LLM backend.
+This will require mounting the repo from step 1 into the docker container and any models you plan to serve.
+
+    For the tensorrtllm_backend repository, you need the following two directories mounted:
+- repo: .../tensorrtllm_backend/:/tensorrtllm_backend
```

Comment:
Could you clarify what repo here means? Does it correspond to `-v /path/to/model/repo:/repo` or `-v /path/to/tensorrtllm_backend:/tensorrtllm_backend` in the command below?

Reply:
Updated to:
- backend: .../tensorrtllm_backend/:/tensorrtllm_backend
- llama repo: .../llama/repo:/Llama-2-7b-hf
- engine: .../tensorrtllm_backend/tensorrt_llm/examples/llama/engine:/engines